packages/angular_devkit/build_angular/src/angular-cli-files/models/es5-polyfills.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import 'core-js/es/symbol';
+import 'core-js/es/function';
+import 'core-js/es/parse-int';
+import 'core-js/es/parse-float';
+import 'core-js/es/number';
+import 'core-js/es/math';
+import 'core-js/es/date';
+import 'core-js/es/regexp';
+import 'core-js/es/map';
+import 'core-js/es/weak-map';
+import 'core-js/es/set';
+import 'core-js/es/promise';
+
+// ES2015 object capabilities
+import 'core-js/es/object/create';
```

Comment:
Why is it necessary to import each function individually?
Why not import everything at once by `import 'core-js/es/object';`?

Reply:
Angular only requires the ES2015 polyfills.  In `core-js` 3.0, `core-js/es/object` includes all current ES polyfills (up to ES2019).  This was an attempt to reduce the bundle size increase.  Using the single imports resulted in an increase of ~15kb to the polyfills file.  Further work is currently being done to attempt to close the gap even further.